src/constants/game/CustomMapExperience.java

```diff
@@ -0,0 +1,21 @@
+package constants.game;
+
+import java.util.HashMap;
+
+public final class CustomMapExperience
+{
+    //Applies experience multiplier to MapID
+    public static final HashMap<Integer, Integer> CUSTOM_MAP_EXPERIENCE = new HashMap<Integer, Integer>() {{
+        put(50000, 20);
+    }};
+
+    public static boolean HasCustomExpValue(int mapId)
+    {
+        return CUSTOM_MAP_EXPERIENCE.containsKey(mapId);
+    }
+
+    public static int GetCustomExpMultiplier(int mapId)
+    {
+        return CUSTOM_MAP_EXPERIENCE.get(mapId);

```

Comment:
If you think about calling another method as a transaction
Picture this
You are the class CustomMapExperience
You are trading (public) GetCustomExpMultiplier for mapId
Someone out there is like "Hey CustomMapExperience, can I GetCustomExpMultiplier for this 5000 (mapId)?"
You're like yeah! and you give him a 5
But the customer is always wrong, so the next person is like "Hey CustomMapExperience, can I getCustomExpMultiplier for this -1 (mapId)
And then you die.